src/screens/Connectors/ConnectorUtils.res

```diff
@@ -1303,3 +1303,11 @@ let getConnectorTypeArrayFromListConnectors = (
     connectorDetail.connector_name->getConnectorNameTypeFromString()
   )
 }
+
+let sortByName = (c1, c2) => {
+  open LogicUtils
+  let getName = connector => connector->getConnectorNameString
+  let nameA = c1->getName
+  let nameB = c2->getName
+  compareLogic(nameB, nameA)->Belt.Int.toFloat
```

Comment:
can we directly pass the value to the compare logic rather than storing the data in nameA and nameB